test/rcheevos/test_consoleinfo.c

```diff
@@ -154,6 +155,7 @@ void test_consoleinfo(void) {
   TEST_PARAMS2(test_memory, RC_CONSOLE_ORIC, 0x010000);
   TEST_PARAMS2(test_memory, RC_CONSOLE_PC8800, 0x011000);
   TEST_PARAMS2(test_memory, RC_CONSOLE_PC_ENGINE, 0x42800);
```

Comment:
This value needs to be updated to reflect the lower amount of RAM exposed for non-CD PCE

Reply:
Changed to `0x02000`